CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ docker run --rm \
   --volume="$PWD:/srv/jekyll" \
   --publish [::1]:4000:4000 \
   jekyll/jekyll \
-  jekyll serve
+  bash -c 'rm -rf ./_site && jekyll serve'
```

Comment:
Following these testing steps as documented worked... but only the first time.

This command leaves behind a `_site/` directory with file permissions showing the default user inside the container as the owner. Where that user differs from the default user on the system where you run `docker run`, the second attempt to run this fails like this:

```text
chown: _site/assets/data: Permission denied
chown: _site/assets/data: Permission denied
chown: _site/assets: Permission denied
chown: _site/assets: Permission denied
chown: _site: Permission denied
chown: _site: Permission denied
```

Removing that directory at startup every time avoids that tiny bit of development friction.